src/renderer/actions/uninstall.js

```diff
@@ -52,23 +53,26 @@ export default async function(config) {
     const sane = doSanityCheck(config);
     if (!sane) return fail();
 
-
+    // Installation channels (such as "stable", "canary") and their latest version's paths.
     const channels = Object.keys(config);
-    const paths = Object.values(config);
+    const discordPaths = Object.values(config);
 
 
-    lognewline("Deleting shims...");
-    const deleteErr = await deleteShims(paths);
+    // Remove BetterDiscord from each Discord version's loader-script.
+    lognewline("Deleting BetterDiscord's loader scripts...");
+    const deleteErr = await deleteShims(discordPaths);
     if (deleteErr) return fail();
-    log("✅ Shims deleted");
+    log("✅ Loader scripts deleted.");
     progress.set(DELETE_SHIM_PROGRESS);
 
 
-    lognewline("Killing Discord...");
+    // Automatically restart Discord clients if they are running.
+    lognewline("Restarting Discord...");
     const killErr = await kill(channels, (RESTART_DISCORD_PROGRESS - progress.value) / channels.length);
-    if (killErr) showRestartNotice(); // No need to bail out
-    else log("✅ Discord restarted");
+    if (killErr) showRestartNotice();  // No need to bail out if we failed, just tell user to restart manually.
```

Comment:
Unnecessary extra space

Reply:
It's a code-style that helps contributors visually see where the statement ends and the comment begins. It helps with skim-reading through a file, no matter if syntax highlighting of comments is a different color or not.

It's for example enshrined in the Python style-guides:

https://www.flake8rules.com/rules/E261.html

So I changed to it everywhere, but I can change back to the tight style if you prefer that. It would be no problem. I just did it to help out readability, for the same reason as the dual linebreaks between functions, which is another thing that really makes code a lot clearer for people who are not yet familiar with a codebase.

Reply:
Thankfully, this is not python and we do not abide by python style guides.